tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -567,30 +567,31 @@ describe('Spectating Rematches', () => {
 
       cy.get('[data-cy=opponent-match-result] [data-cy-result-img=lost]').should('be.visible');
 
+      // The stalemate dialog should dissappear
+      cy.wait(1000);
+      cy.get('#opponent-requested-stalemate-dialog').should('not.exist');
```

Comment:
When you want to ensure that something disappears, it's sometimes necessary to `cy.wait()` first just to make sure that we are checking that the thing is not present after a delay, instead of in the beginning

Reply:
On second thought I think we can remove this since the element was already present in the previous update/assertion so we don't need to worry about it passing the not exists assertion by checking before the overlay renders since it's already rendered